### PR TITLE
Update members 2021

### PR DESCRIPTION
**Pull request by elit-altum**
- Add new page for Alumni members
image

- Added all new members & leads for the batch of 2020-2021

### CODE REVIEW

File: src/contents/members/anirudh.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-username:  firefinchdev
-name:  Anirudh Gupta
-designation: Android Co-Lead
+username: firefinchdev
+name: Anirudh Gupta
+designation: Android
```
Comment (mannprerak2) [inline src/contents/members/anirudh.md]: He's also in batch_2020

File: src/contents/members/ashray.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
 username: 14ashray
 name: Ashray Aggarwal
-designation: DSC Co-Lead, ML
+designation: DSC Co-Lead
```
Comment (mannprerak2) [inline src/contents/members/ashray.md:4]: batch2020

File: src/contents/members/kunalSinha.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+username:
+name: Kunal Sinha
+designation: Graphic Designer
```
Comment (mannprerak2) [inline src/contents/members/kunalSinha.md]: Nontech lead

**Review by mannprerak2 — CHANGES_REQUESTED** [review]

File: src/contents/members/kunalSinha.md (hunk 3 above)
Comment (elit-altum) [reply src/contents/members/kunalSinha.md]: Should I then remove @priyanka2399  from the Non-Tech team ?

File: src/contents/members/kunalSinha.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
 username:
 name: Kunal Sinha
-designation: Graphic Designer
+designation: Non-Tech Lead & Graphic Designer
```
Comment (mannprerak2) [inline src/contents/members/kunalSinha.md]: Remove graphic designer, sorry I didn't inform this before

**Comment by elit-altum** [comment]
@mannprerak2 good to merge!

**Pull request merged**